cmd/rofl/init_artifacts/compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  mycontainer:
+    build: .
+    image: "docker.io/org_name/image_name" # Always provide FQDN image names!
+    platform: linux/amd64
+    volumes:
+      - /storage/my_container:/root/.my_volume  # Persistent per-machine storage.
```

Comment:
This is not the best. Please see how to declare volumes correctly, eg. you should not use absolute paths but rather declare volumes explicitly.

Looking at docs it seems someone changed this as I remember I wrote a section on how to do this correctly.

Reply:
I see docs about persistent volumes were changed in https://github.com/oasisprotocol/oasis-sdk/commit/c0c6a5bf765fd5775dd577bd558a66c3968c1dd6 and I didn't notice that part.

We should change it back to use volumes without referencing `/storage` directly IMO? At least that seems the most correct approach to me.

Reply:
![image](https://github.com/user-attachments/assets/6955c347-1406-4b98-b971-6b0e5a956a71)

Reply:
I agree. This approach is also cleaner because it uses preconfigured `/storage` path and it can be recycled in other docker images.